templates/CRM/Contact/Form/Contact.tpl

```diff
@@ -326,6 +332,14 @@
         if (checkNum < runningCheck) {
           response.reject();
         } else {
+          $.each(data['values'], function(cid, result) {
+            if ( $.inArray(cid, dismissed_matches) == -1 ) {
+              dismissed_matches.push(cid);
+            } else {
+              delete data.values[cid];
+              data.count--;
+            }
+          });
```

Comment:
My assumption here is that for contacts already matched the user has dismissed them.  Otherwise they would click on the duplicate contact and leave the form.  

I was also thinking the matched contacts should be submitted so validation can skip them when as duplicates when the Supervised rule is called during save.